rma/models/rma_order_line.py

```diff
@@ -86,9 +84,11 @@ def _compute_qty_to_receive(self):
         for rec in self:
             rec.qty_to_receive = 0.0
             if rec.receipt_policy == 'ordered':
-                rec.qty_to_receive = rec.product_qty - rec.qty_received
+                rec.qty_to_receive = \
+                    rec.product_qty - rec.qty_incoming -rec.qty_received
             elif rec.receipt_policy == 'delivered':
-                rec.qty_to_receive = rec.qty_delivered - rec.qty_received
+                rec.qty_to_receive = \
+                    rec.qty_delivered - rec.qty_incoming - rec.qty_received
```

Comment:
We have to do the same in the _compute_qty_to_receive method with the qty_outgoing. I write this comment here as I reminder for myself :smile: